sdk/js/src/relayer/structs.ts

```diff
@@ -41,12 +41,31 @@ export function parseRefundStatus(index: number) {
     : RefundStatus.CrossChainRefundFailProviderNotSupported;
 }
 
+export enum KeyType {
+  VAA = 1,
+  CCTP = 2,
+}
+export interface MessageKey {
+  keyType: KeyType | number;
+  key: ethers.BytesLike;
+}
+
 export interface VaaKey {
   chainId: number;
   emitterAddress: Buffer;
   sequence: BigNumber;
 }
 
+export interface CCTPKey {
+  domain: number;
```

Comment:
just curious, what does domain represent in this context?

Reply:
Domain is Circle's version of chainId